src/app/core/auth/directives/is-authenticated.directive.ts

```diff
@@ -15,12 +15,11 @@ export class IsAuthenticatedDirective extends AbstractIfThenElseDirective implem
 	private _isAuthenticated = true;
 
 	constructor(
-		viewContainer: ViewContainerRef,
 		templateRef: TemplateRef<any>,
 		private sessionService: SessionService,
```

Comment:
For all of these classes that extend from `AbstractIfThenElseDirective`, I choose not to use `inject` for `viewContainer`.  The reasoning was that I was unsure how it would interact in the case that the thenTemplate is overridden like in the example below.

```html
<div *isAuthenticated="true; then thenBlock"></div>
<ng-template #thenBlock>...</ng-template>
```